api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/ConceptTranslatorImpl.java

```diff
@@ -103,11 +106,11 @@ private void addConceptCoding(Coding coding, String system, String code, Concept
 		coding.setSystem(system);
 		coding.setCode(code);
 		ConceptName conceptName = concept.getName(userDefaultProperties.getDefaultLocale());
-		if (conceptName.getName() == null) {
+		if (conceptName == null || conceptName.getName() == null) {
 			conceptName = concept.getName();
 		}
 		
-		String display = conceptName.getName() == null ? "" : conceptName.getName();
+		String display = (conceptName == null || conceptName.getName() == null) ? StringUtils.EMPTY : conceptName.getName();
```

Comment:
It seems kind of silly to import `StringUtils` just to get a reference to the empty string.

```suggestion
		String display = (conceptName == null || conceptName.getName() == null) ? "" : conceptName.getName();
```